cmd/image-builder-tests/main_test.go

```diff
@@ -106,6 +105,42 @@ func RunTestWithClient(t *testing.T, ib string)  {
 	require.NoError(t, err)
 	require.Contains(t, []string{"pending", "running"}, composeStatus.ImageStatus.Status)
 
+	// A compose with gcp as a target
+	composeRequest = server.ComposeRequest{
+		Distribution: distro,
+		ImageRequests: []server.ImageRequest{
+			server.ImageRequest{
+				Architecture: arch,
+				ImageType: architectures[0].ImageTypes[0],
```

Comment:
My assumption is that this is the distro-specific list of supported architectures and image types. So from looking at `internal/distro/rhel84/distro.go` from `osbuild-composer`, the `[0]` arch is x86_64, which is fine, but the Image type `[0]` is AMI. This should be changed to VHD or VMDK image types for now.

Reply:
ah good point